lib/wallet/src/Cardano/Api/Gen.hs

```diff
@@ -1057,38 +1058,17 @@ genCostModels = do
 genExecutionUnitPrices :: Gen ExecutionUnitPrices
 genExecutionUnitPrices = ExecutionUnitPrices <$> genRational <*> genRational
 
-genProtocolParameters :: Gen ProtocolParameters
-genProtocolParameters =
-  ProtocolParameters
-    <$> ((,) <$> genNat <*> genNat)
-    <*> (Just <$> genRational)
-    <*> liftArbitrary genPraosNonce
-    <*> genNat
-    <*> genNat
-    <*> genNat
-    <*> genNat
-    <*> genNat
-    <*> liftArbitrary genLovelace
-    <*> genLovelace
-    <*> genLovelace
-    <*> genLovelace
-    <*> genEpochNo
-    <*> genNat
-    <*> genRationalInt64
-    <*> genRational
-    <*> genRational
-    <*> liftArbitrary genLovelace
-    <*> genCostModels
-    <*> liftArbitrary genExecutionUnitPrices
-    <*> liftArbitrary genExecutionUnits
-    <*> liftArbitrary genExecutionUnits
-    <*> liftArbitrary genNat
-    <*> liftArbitrary genNat
-    <*> liftArbitrary genNat
-    <*> liftArbitrary genLovelace
-
-genProtocolParametersWithAlonzoScripts :: Gen ProtocolParameters
-genProtocolParametersWithAlonzoScripts =
+-- | Dummy value for suitable for being included in the pre-image of the script
+-- integrity hash.
+{-# NOINLINE protocolParametersForHashing #-}
+protocolParametersForHashing :: ProtocolParameters
+protocolParametersForHashing = generateWith (GenSeed 0) genSizeDefault
+    genRecentEraProtocolParameters
```

Comment:
Just a suggestion, but maybe this is a bit easier to read:
```suggestion
protocolParametersForHashing =
    generateWith (GenSeed 0) genSizeDefault genRecentEraProtocolParameters
```

Reply:
Having it as 

```
protocolParametersForHashing =
    generateWith (GenSeed 0) genSizeDefault
        genRecentEraProtocolParameters
```
in that case (still having genRecentEraProtocolParameters indented one step more relative to generateWith)